.github/workflows/build-and-deploy.yml

```diff
@@ -69,6 +69,39 @@ jobs:
           message: |
             Review app deployed to <${{ steps.deploy.outputs.environment_url }}>
 
+  deploy_review_app_aks:
+    name: Deploy to review environment for AKS
+    runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'deploy-aks')
+    concurrency: deploy_review_${{ github.event.pull_request.number }}
+    needs: [build_image]
+    environment:
+      name: aks-review
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Deploy to AKS
+        uses: ./.github/actions/deploy-environment-aks
+        id: deploy_aks
+        with:
+          environment: aks-review
+          image-tag: ${{ github.sha }}
```

Comment:
for consistency might be better to use ${{ needs.build_image.outputs.image_name_tag }}

Reply:
This is consistent with the approach thats currently set, for the standard review app deploy the image-tag: ${{ github.sha }}, if it's set to ${{ needs.build_image.outputs.image_name_tag }} then it will append the image reference incorrectly as that's used for the image_name_tag which is the first part of the reference i believe

 image_name_tag: ${{ needs.build_image.outputs.image_name_tag }}

Reply:
Eventually we should replace ./.github/actions/build-image with https://github.com/DFE-Digital/github-actions/tree/master/build-docker-image and it would output the single tag. We can keep it simple for now and use ${{ github.sha }}